Dockerfile

```diff
@@ -189,6 +192,9 @@ RUN echo "source /.version_information.sh" >> ~/.bashrc
 COPY .version_information.sh /
 
 # container startup setup
+ENV WORKSPACE=/docker-ros/ws
+WORKDIR $WORKSPACE
+RUN mkdir src
```

Comment:
```suggestion
```

I would suggest to create the `src`  folder only when it is explicitly needed (e.g. by mounting code). This prevents the folder from being empty in images that may only contain compiled code in the workspace.